From 050ee3a7cb801fade428cc7b63585333c377fac5 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:54 -0400
Subject: [redhat] kabi: Adding symbol kill_pid (kernel/signal.c)

Message-id: <b0e18d632f3f5fb6eae1247401eafa1a7b2cd205.1410510773.git.poros@redhat.com>
Patchwork-id: 95939
O-Subject: [PATCH RHEL7.1 114/233] kabi: Adding symbol kill_pid (kernel/signal.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    GAB kills unresponsive userland clients.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/kill_pid
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/kill_pid
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/kill_pid

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/kill_pid b/redhat/kabi/kabi-module/kabi_ppc64/kill_pid
new file mode 100644
index 0000000..0f33ec3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/kill_pid
@@ -0,0 +1,2 @@
+#1-
+0x589718e5 kill_pid vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/kill_pid b/redhat/kabi/kabi-module/kabi_s390x/kill_pid
new file mode 100644
index 0000000..537b4f8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/kill_pid
@@ -0,0 +1,2 @@
+#1-
+0x413fd599 kill_pid vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/kill_pid b/redhat/kabi/kabi-module/kabi_x86_64/kill_pid
new file mode 100644
index 0000000..0f48bba
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/kill_pid
@@ -0,0 +1,2 @@
+#1-
+0x52fd15e1 kill_pid vmlinux EXPORT_SYMBOL
-- 
1.7.1